url.bs

```diff
@@ -583,7 +583,7 @@ decisions made, i.e. whether to use the <a>same site</a> or <a>schemelessly same
 
   <p class=note>If <var>beStrict</var> is false, <var>domain</var> is an <a>ASCII string</a>, and
   <a>strictly splitting</a> <var>domain</var> on U+002E (.) does not produce any
-  <a for=list>item</a> that <a for=string>starts with</a> "<code>xn--</code>", this step is
+  <a for=list>item</a> that <a for=string>starts with</a> an <a>ASCII case-insensitive</a> match for "<code>xn--</code>", this step is
   equivalent to <a>ASCII lowercasing</a> <var>domain</var>.
```

Comment:
Needs a wrapping fix:
```suggestion
  <a for=list>item</a> that <a for=string>starts with</a> an <a>ASCII case-insensitive</a> match for 
  "<code>xn--</code>", this step is equivalent to <a>ASCII lowercasing</a> <var>domain</var>.
```
Although this is still not entirely technically correct given the signature of "starts with", I think it's good enough?

Reply:
@annevk  Yeah the "starts with" thing is a bit awkward, but I couldn't find anything better in infra, nor could I think of any way it could be ambiguous as written, so... 🤷‍♂️ .